FOP integration tests fail in native mode with GraalVM 20.3.0

Quarkus 1.11.0.CR1 bumped GraalVM to 20.3.0. We now see a failure with FOP:

```
2021-01-07T11:59:16.4618152Z [INFO] -------------------------------------------------------
2021-01-07T11:59:16.4621400Z [INFO]  T E S T S
2021-01-07T11:59:16.4627367Z [INFO] -------------------------------------------------------
2021-01-07T11:59:17.5989577Z [INFO] Running org.apache.camel.quarkus.component.fop.it.FopIT
2021-01-07T11:59:17.7476077Z ======================================================================================
2021-01-07T11:59:17.7480519Z   native.image.path was not set, making a guess for the correct path of native image
2021-01-07T11:59:17.7487405Z   guessed path: /home/runner/work/camel-quarkus/camel-quarkus/integration-tests/fop/target/camel-quarkus-integration-test-fop-1.6.0-SNAPSHOT-runner
2021-01-07T11:59:17.7492276Z ======================================================================================
2021-01-07T11:59:17.7504684Z Executing [/home/runner/work/camel-quarkus/camel-quarkus/integration-tests/fop/target/camel-quarkus-integration-test-fop-1.6.0-SNAPSHOT-runner, -Dquarkus.http.port=42339, -Dquarkus.http.ssl-port=41935, -Dtest.url=http://localhost:42339, -Dquarkus.log.file.path=target/target/quarkus.log]
2021-01-07T11:59:17.7702410Z 2021-01-07 11:59:17,769 INFO  [org.apa.cam.qua.cor.CamelBootstrapRecorder] (main) bootstrap runtime: org.apache.camel.quarkus.core.CamelContextRuntime
2021-01-07T11:59:17.7715417Z 2021-01-07 11:59:17,770 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 3.7.0 (camel-1) is starting
2021-01-07T11:59:17.7726193Z 2021-01-07 11:59:17,771 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) StreamCaching is not in use. If using streams then it's recommended to enable stream caching. See more details at http://camel.apache.org/stream-caching.html
2021-01-07T11:59:17.7734290Z 2021-01-07 11:59:17,772 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Total 0 routes, of which 0 are started
2021-01-07T11:59:17.7741673Z 2021-01-07 11:59:17,773 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 3.7.0 (camel-1) started in 2ms
2021-01-07T11:59:17.7835149Z 2021-01-07 11:59:17,782 INFO  [io.quarkus] (main) camel-quarkus-integration-test-fop 1.6.0-SNAPSHOT native (powered by Quarkus 1.11.0.CR1) started in 0.025s. Listening on: http://0.0.0.0:42339
2021-01-07T11:59:17.7842181Z 2021-01-07 11:59:17,782 INFO  [io.quarkus] (main) Profile prod activated. 
2021-01-07T11:59:17.7844430Z 2021-01-07 11:59:17,782 INFO  [io.quarkus] (main) Installed features: [camel-core, camel-fop, camel-support-common, camel-support-commons-logging, camel-xml-jaxp, cdi, resteasy]
2021-01-07T11:59:19.9563607Z 2021-01-07 11:59:19,955 INFO  [org.apa.cam.qua.com.fop.it.FopResource] (executor-thread-1) Sending to fop: <fo:root xmlns:fo="http://www.w3.org/1999/XSL/Format">
2021-01-07T11:59:19.9565806Z   <fo:layout-master-set>
2021-01-07T11:59:19.9566960Z     <fo:simple-page-master master-name="only">
2021-01-07T11:59:19.9568308Z       <fo:region-body region-name="xsl-region-body" margin="0.7in"  padding="0" />
2021-01-07T11:59:19.9569739Z       <fo:region-before region-name="xsl-region-before" extent="0.7in" />
2021-01-07T11:59:19.9571474Z         <fo:region-after region-name="xsl-region-after" extent="0.7in" />
2021-01-07T11:59:19.9572729Z       </fo:simple-page-master>
2021-01-07T11:59:19.9573744Z     </fo:layout-master-set>
2021-01-07T11:59:19.9583457Z     <fo:page-sequence master-reference="only">
2021-01-07T11:59:19.9584813Z       <fo:flow flow-name="xsl-region-body">
2021-01-07T11:59:19.9585928Z       <fo:block font-family="Freedom">hello</fo:block>
2021-01-07T11:59:19.9586694Z     </fo:flow>
2021-01-07T11:59:19.9587503Z   </fo:page-sequence>
2021-01-07T11:59:19.9588160Z </fo:root>
2021-01-07T11:59:19.9644419Z 2021-01-07 11:59:19,963 WARN  [org.apa.fop.fo.pro.CommonHyphenation] (executor-thread-1) Substituted specified hyphenation character (0x2d) with 0x20 because the font doesn't have the specified hyphenation character: Freedom,normal,400
2021-01-07T11:59:19.9662763Z 2021-01-07 11:59:19,964 ERROR [org.apa.fop.are.AreaTreeModel] (executor-thread-1) Error while rendering page 1: java.lang.UnsupportedOperationException: Not implemented yet for GraalVM native images
2021-01-07T11:59:19.9666199Z 	at sun.java2d.cmm.lcms.LCMS.loadProfileNative(LCMS.java:13)
2021-01-07T11:59:19.9667800Z 	at sun.java2d.cmm.lcms.LCMS.loadProfile(LCMS.java:42)
2021-01-07T11:59:19.9669091Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:790)
2021-01-07T11:59:19.9670279Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:1026)
2021-01-07T11:59:19.9671467Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:991)
2021-01-07T11:59:19.9672533Z 	at java.awt.color.ICC_Profile$2.run(ICC_Profile.java:940)
2021-01-07T11:59:19.9673525Z 	at java.awt.color.ICC_Profile$2.run(ICC_Profile.java:936)
2021-01-07T11:59:19.9675006Z 	at java.security.AccessController.doPrivileged(AccessController.java:84)
2021-01-07T11:59:19.9676722Z 	at java.awt.color.ICC_Profile.getStandardProfile(ICC_Profile.java:935)
2021-01-07T11:59:19.9678163Z 	at java.awt.color.ICC_Profile.getDeferredInstance(ICC_Profile.java:1067)
2021-01-07T11:59:19.9679478Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:854)
2021-01-07T11:59:19.9680833Z 	at java.awt.color.ColorSpace.getInstance(ColorSpace.java:298)
2021-01-07T11:59:19.9682164Z 	at java.awt.Color.getColorSpace(Color.java:1183)
2021-01-07T11:59:19.9684117Z 	at org.apache.fop.pdf.PDFColorHandler.establishColorFromColor(PDFColorHandler.java:95)
2021-01-07T11:59:19.9686662Z 	at org.apache.fop.pdf.PDFColorHandler.establishColor(PDFColorHandler.java:88)
2021-01-07T11:59:19.9689128Z 	at org.apache.fop.render.pdf.PDFContentGenerator.setColor(PDFContentGenerator.java:458)
2021-01-07T11:59:19.9693515Z 	at org.apache.fop.render.pdf.PDFContentGenerator.setColor(PDFContentGenerator.java:435)
2021-01-07T11:59:19.9696044Z 	at org.apache.fop.render.pdf.PDFContentGenerator.setColor(PDFContentGenerator.java:460)
2021-01-07T11:59:19.9698689Z 	at org.apache.fop.render.pdf.PDFContentGenerator.updateColor(PDFContentGenerator.java:483)
2021-01-07T11:59:19.9701089Z 	at org.apache.fop.render.pdf.PDFPainter.drawText(PDFPainter.java:428)
2021-01-07T11:59:19.9702902Z 	at org.apache.fop.render.pdf.PDFPainter.drawText(PDFPainter.java:449)
2021-01-07T11:59:19.9704775Z 	at org.apache.fop.render.intermediate.IFRenderer$TextUtil.flush(IFRenderer.java:1238)
2021-01-07T11:59:19.9706987Z 	at org.apache.fop.render.intermediate.IFRenderer.renderText(IFRenderer.java:1064)
2021-01-07T11:59:19.9709442Z 	at org.apache.fop.render.AbstractRenderer.renderInlineArea(AbstractRenderer.java:861)
2021-01-07T11:59:19.9712120Z 	at org.apache.fop.render.intermediate.IFRenderer.renderInlineArea(IFRenderer.java:927)
2021-01-07T11:59:19.9714726Z 	at org.apache.fop.render.AbstractRenderer.renderLineArea(AbstractRenderer.java:845)
2021-01-07T11:59:19.9717091Z 	at org.apache.fop.render.AbstractRenderer.renderBlocks(AbstractRenderer.java:704)
2021-01-07T11:59:19.9719339Z 	at org.apache.fop.render.AbstractRenderer.renderBlock(AbstractRenderer.java:758)
2021-01-07T11:59:19.9721700Z 	at org.apache.fop.render.intermediate.IFRenderer.renderBlock(IFRenderer.java:999)
2021-01-07T11:59:19.9724260Z 	at org.apache.fop.render.AbstractRenderer.renderBlocks(AbstractRenderer.java:689)
2021-01-07T11:59:19.9726530Z 	at org.apache.fop.render.AbstractRenderer.renderFlow(AbstractRenderer.java:591)
2021-01-07T11:59:19.9734269Z 	at org.apache.fop.render.AbstractPathOrientedRenderer.renderFlow(AbstractPathOrientedRenderer.java:789)
2021-01-07T11:59:19.9737329Z 	at org.apache.fop.render.AbstractRenderer.renderMainReference(AbstractRenderer.java:568)
2021-01-07T11:59:19.9996636Z 	at org.apache.fop.render.AbstractRenderer.renderBodyRegion(AbstractRenderer.java:433)
2021-01-07T11:59:20.0000017Z 	at org.apache.fop.render.AbstractRenderer.renderRegionViewport(AbstractRenderer.java:380)
2021-01-07T11:59:20.0003198Z 	at org.apache.fop.render.intermediate.IFRenderer.renderRegionViewport(IFRenderer.java:738)
2021-01-07T11:59:20.0006194Z 	at org.apache.fop.render.AbstractRenderer.renderPageAreas(AbstractRenderer.java:345)
2021-01-07T11:59:20.0008738Z 	at org.apache.fop.render.AbstractRenderer.renderPage(AbstractRenderer.java:318)
2021-01-07T11:59:20.0011224Z 	at org.apache.fop.render.intermediate.IFRenderer.renderPage(IFRenderer.java:587)
2021-01-07T11:59:20.0013604Z 	at org.apache.fop.area.RenderPagesModel.addPage(RenderPagesModel.java:123)
2021-01-07T11:59:20.0016862Z 	at org.apache.fop.layoutmgr.AbstractPageSequenceLayoutManager.finishPage(AbstractPageSequenceLayoutManager.java:316)
2021-01-07T11:59:20.0023562Z 	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.finishPage(PageSequenceLayoutManager.java:243)
2021-01-07T11:59:20.0027668Z 	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.activateLayout(PageSequenceLayoutManager.java:147)
2021-01-07T11:59:20.0030611Z 	at org.apache.fop.area.AreaTreeHandler.endPageSequence(AreaTreeHandler.java:267)
2021-01-07T11:59:20.0032831Z 	at org.apache.fop.fo.pagination.PageSequence.endOfNode(PageSequence.java:139)
2021-01-07T11:59:20.0034799Z 	at org.apache.fop.fo.FOTreeBuilder$MainFOHandler.endElement(FOTreeBuilder.java:362)
2021-01-07T11:59:20.0041149Z 	at org.apache.fop.fo.FOTreeBuilder.endElement(FOTreeBuilder.java:190)
2021-01-07T11:59:20.0051755Z 	at com.sun.org.apache.xml.internal.serializer.ToXMLSAXHandler.endElement(ToXMLSAXHandler.java:263)
2021-01-07T11:59:20.0056924Z 	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:610)
2021-01-07T11:59:20.0061552Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1718)
2021-01-07T11:59:20.0066395Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2883)
2021-01-07T11:59:20.0070358Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
2021-01-07T11:59:20.0073880Z 	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
2021-01-07T11:59:20.0078235Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
2021-01-07T11:59:20.0082312Z 	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
2021-01-07T11:59:20.0085460Z 	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
2021-01-07T11:59:20.0088251Z 	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
2021-01-07T11:59:20.0091083Z 	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
2021-01-07T11:59:20.0093948Z 	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
2021-01-07T11:59:20.0140882Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transformIdentity(TransformerImpl.java:660)
2021-01-07T11:59:20.0147765Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:774)
2021-01-07T11:59:20.0151333Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:370)
2021-01-07T11:59:20.0154094Z 	at org.apache.camel.component.fop.FopProducer.transform(FopProducer.java:98)
2021-01-07T11:59:20.0156214Z 	at org.apache.camel.component.fop.FopProducer.process(FopProducer.java:66)
2021-01-07T11:59:20.0159298Z 	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66)
2021-01-07T11:59:20.0163123Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:218)
2021-01-07T11:59:20.0166713Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor$1.process(SharedCamelInternalProcessor.java:112)
2021-01-07T11:59:20.0170398Z 	at org.apache.camel.impl.engine.DefaultAsyncProcessorAwaitManager.process(DefaultAsyncProcessorAwaitManager.java:83)
2021-01-07T11:59:20.0174304Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:109)
2021-01-07T11:59:20.0177464Z 	at org.apache.camel.support.cache.DefaultProducerCache.send(DefaultProducerCache.java:189)
2021-01-07T11:59:20.0180270Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:176)
2021-01-07T11:59:20.0249943Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:172)
2021-01-07T11:59:20.0253432Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.requestBody(DefaultProducerTemplate.java:413)
2021-01-07T11:59:20.0256482Z 	at org.apache.camel.quarkus.component.fop.it.FopResource.post(FopResource.java:55)
2021-01-07T11:59:20.0259013Z 	at org.apache.camel.quarkus.component.fop.it.FopResource_ClientProxy.post(FopResource_ClientProxy.zig:161)
2021-01-07T11:59:20.0260983Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2021-01-07T11:59:20.0262649Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:170)
2021-01-07T11:59:20.0264881Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
2021-01-07T11:59:20.0267815Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:643)
2021-01-07T11:59:20.0271550Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:507)
2021-01-07T11:59:20.0341564Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:457)
2021-01-07T11:59:20.0345467Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.0349509Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:459)
2021-01-07T11:59:20.0352282Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:419)
2021-01-07T11:59:20.0354813Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:393)
2021-01-07T11:59:20.0357314Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:68)
2021-01-07T11:59:20.0359807Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:492)
2021-01-07T11:59:20.0362298Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
2021-01-07T11:59:20.0364866Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
2021-01-07T11:59:20.0368653Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.0372524Z 	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
2021-01-07T11:59:20.0375099Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
2021-01-07T11:59:20.0377891Z 	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:73)
2021-01-07T11:59:20.0454777Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:138)
2021-01-07T11:59:20.0458228Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.access$000(VertxRequestHandler.java:41)
2021-01-07T11:59:20.0461461Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:93)
2021-01-07T11:59:20.0463850Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
2021-01-07T11:59:20.0465841Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2021-01-07T11:59:20.0467817Z 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
2021-01-07T11:59:20.0470171Z 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
2021-01-07T11:59:20.0471915Z 	at java.lang.Thread.run(Thread.java:834)
2021-01-07T11:59:20.0472948Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2021-01-07T11:59:20.0474814Z 	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
2021-01-07T11:59:20.0477778Z 	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
2021-01-07T11:59:20.0479572Z 
2021-01-07T11:59:20.0540883Z 2021-01-07 11:59:19,974 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-1) HTTP Request to /fop/post?userConfigURL=file%3A%2F%2F%2Fhome%2Frunner%2Fwork%2Fcamel-quarkus%2Fcamel-quarkus%2Fintegration-tests%2Ffop%2Ftarget%2Ftmp%2Fmycfg.xml failed, error id: a430f80f-fc91-40d6-b7d0-f71c6e7a334c-1: org.jboss.resteasy.spi.UnhandledException: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[]
2021-01-07T11:59:20.0545722Z 	at org.jboss.resteasy.core.ExceptionHandler.handleApplicationException(ExceptionHandler.java:106)
2021-01-07T11:59:20.0548679Z 	at org.jboss.resteasy.core.ExceptionHandler.handleException(ExceptionHandler.java:372)
2021-01-07T11:59:20.0551484Z 	at org.jboss.resteasy.core.SynchronousDispatcher.writeException(SynchronousDispatcher.java:218)
2021-01-07T11:59:20.0554203Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:519)
2021-01-07T11:59:20.0556712Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
2021-01-07T11:59:20.0559264Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
2021-01-07T11:59:20.0563063Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.0566903Z 	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
2021-01-07T11:59:20.0569512Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
2021-01-07T11:59:20.0572293Z 	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:73)
2021-01-07T11:59:20.0575561Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:138)
2021-01-07T11:59:20.0578904Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.access$000(VertxRequestHandler.java:41)
2021-01-07T11:59:20.0584508Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:93)
2021-01-07T11:59:20.0586897Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
2021-01-07T11:59:20.0588886Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2021-01-07T11:59:20.0590833Z 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
2021-01-07T11:59:20.0593048Z 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
2021-01-07T11:59:20.0594772Z 	at java.lang.Thread.run(Thread.java:834)
2021-01-07T11:59:20.0595785Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2021-01-07T11:59:20.0597663Z 	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
2021-01-07T11:59:20.0600796Z 	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
2021-01-07T11:59:20.0603513Z Caused by: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[]
2021-01-07T11:59:20.0605672Z 	at org.apache.camel.support.DefaultExchange.setException(DefaultExchange.java:425)
2021-01-07T11:59:20.0608879Z 	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:69)
2021-01-07T11:59:20.0612696Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:218)
2021-01-07T11:59:20.0616179Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor$1.process(SharedCamelInternalProcessor.java:112)
2021-01-07T11:59:20.0619850Z 	at org.apache.camel.impl.engine.DefaultAsyncProcessorAwaitManager.process(DefaultAsyncProcessorAwaitManager.java:83)
2021-01-07T11:59:20.0623932Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:109)
2021-01-07T11:59:20.0627111Z 	at org.apache.camel.support.cache.DefaultProducerCache.send(DefaultProducerCache.java:189)
2021-01-07T11:59:20.0629911Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:176)
2021-01-07T11:59:20.0632772Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:172)
2021-01-07T11:59:20.0635881Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.requestBody(DefaultProducerTemplate.java:413)
2021-01-07T11:59:20.0638799Z 	at org.apache.camel.quarkus.component.fop.it.FopResource.post(FopResource.java:55)
2021-01-07T11:59:20.0641333Z 	at org.apache.camel.quarkus.component.fop.it.FopResource_ClientProxy.post(FopResource_ClientProxy.zig:161)
2021-01-07T11:59:20.0643144Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2021-01-07T11:59:20.0644803Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:170)
2021-01-07T11:59:20.0647041Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
2021-01-07T11:59:20.0649956Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:643)
2021-01-07T11:59:20.0653701Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:507)
2021-01-07T11:59:20.0656988Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:457)
2021-01-07T11:59:20.0741445Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.0745573Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:459)
2021-01-07T11:59:20.0748365Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:419)
2021-01-07T11:59:20.0750869Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:393)
2021-01-07T11:59:20.0753668Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:68)
2021-01-07T11:59:20.0756159Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:492)
2021-01-07T11:59:20.0757583Z 	... 17 more
2021-01-07T11:59:20.0760099Z Caused by: com.oracle.svm.core.jdk.UnsupportedFeatureError: Resource bundle lookup must be loaded during native image generation: com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMessages
2021-01-07T11:59:20.0763439Z 	at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:87)
2021-01-07T11:59:20.0765342Z 	at java.util.ListResourceBundle.loadLookup(ListResourceBundle.java:108)
2021-01-07T11:59:20.0767261Z 	at java.util.ListResourceBundle.handleGetObject(ListResourceBundle.java:130)
2021-01-07T11:59:20.0768995Z 	at java.util.ResourceBundle.getObject(ResourceBundle.java:555)
2021-01-07T11:59:20.0770430Z 	at java.util.ResourceBundle.getString(ResourceBundle.java:521)
2021-01-07T11:59:20.0773271Z 	at com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMsg.getErrorMessage(ErrorMsg.java:311)
2021-01-07T11:59:20.0776805Z 	at com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMsg.toString(ErrorMsg.java:284)
2021-01-07T11:59:20.0778854Z 	at java.lang.String.valueOf(String.java:2951)
2021-01-07T11:59:20.0779842Z 	at java.io.PrintStream.println(PrintStream.java:897)
2021-01-07T11:59:20.0782064Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.error(TransformerImpl.java:1376)
2021-01-07T11:59:20.0785680Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.postErrorToListener(TransformerImpl.java:841)
2021-01-07T11:59:20.0789502Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:782)
2021-01-07T11:59:20.0792816Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:370)
2021-01-07T11:59:20.0795539Z 	at org.apache.camel.component.fop.FopProducer.transform(FopProducer.java:98)
2021-01-07T11:59:20.0797661Z 	at org.apache.camel.component.fop.FopProducer.process(FopProducer.java:66)
2021-01-07T11:59:20.0800709Z 	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66)
2021-01-07T11:59:20.0802926Z 	... 39 more
2021-01-07T11:59:20.0803185Z 
2021-01-07T11:59:20.1360165Z 2021-01-07 11:59:20,130 INFO  [org.apa.cam.qua.com.fop.it.FopResource] (executor-thread-1) Sending to fop: <fo:root xmlns:fo="http://www.w3.org/1999/XSL/Format">
2021-01-07T11:59:20.1362155Z   <fo:layout-master-set>
2021-01-07T11:59:20.1363341Z     <fo:simple-page-master master-name="only">
2021-01-07T11:59:20.1364688Z       <fo:region-body region-name="xsl-region-body" margin="0.7in"  padding="0" />
2021-01-07T11:59:20.1366123Z       <fo:region-before region-name="xsl-region-before" extent="0.7in" />
2021-01-07T11:59:20.1367537Z         <fo:region-after region-name="xsl-region-after" extent="0.7in" />
2021-01-07T11:59:20.1368754Z       </fo:simple-page-master>
2021-01-07T11:59:20.1369761Z     </fo:layout-master-set>
2021-01-07T11:59:20.1370884Z     <fo:page-sequence master-reference="only">
2021-01-07T11:59:20.1372009Z       <fo:flow flow-name="xsl-region-body">
2021-01-07T11:59:20.1372820Z       <fo:block>hello</fo:block>
2021-01-07T11:59:20.1373443Z     </fo:flow>
2021-01-07T11:59:20.1374266Z   </fo:page-sequence>
2021-01-07T11:59:20.1374922Z </fo:root>
2021-01-07T11:59:20.1377211Z 2021-01-07 11:59:20,133 ERROR [org.apa.fop.are.AreaTreeModel] (executor-thread-1) Error while rendering page 1: java.lang.UnsupportedOperationException: Not implemented yet for GraalVM native images
2021-01-07T11:59:20.1379609Z 	at sun.java2d.cmm.lcms.LCMS.loadProfileNative(LCMS.java:13)
2021-01-07T11:59:20.1382319Z 	at sun.java2d.cmm.lcms.LCMS.loadProfile(LCMS.java:42)
2021-01-07T11:59:20.1383774Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:790)
2021-01-07T11:59:20.1384995Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:1026)
2021-01-07T11:59:20.1386236Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:991)
2021-01-07T11:59:20.1387340Z 	at java.awt.color.ICC_Profile$2.run(ICC_Profile.java:940)
2021-01-07T11:59:20.1388316Z 	at java.awt.color.ICC_Profile$2.run(ICC_Profile.java:936)
2021-01-07T11:59:20.1389823Z 	at java.security.AccessController.doPrivileged(AccessController.java:84)
2021-01-07T11:59:20.1391523Z 	at java.awt.color.ICC_Profile.getStandardProfile(ICC_Profile.java:935)
2021-01-07T11:59:20.1392956Z 	at java.awt.color.ICC_Profile.getDeferredInstance(ICC_Profile.java:1067)
2021-01-07T11:59:20.1394285Z 	at java.awt.color.ICC_Profile.getInstance(ICC_Profile.java:854)
2021-01-07T11:59:20.1395645Z 	at java.awt.color.ColorSpace.getInstance(ColorSpace.java:298)
2021-01-07T11:59:20.1396975Z 	at java.awt.Color.getColorSpace(Color.java:1183)
2021-01-07T11:59:20.1398933Z 	at org.apache.fop.pdf.PDFColorHandler.establishColorFromColor(PDFColorHandler.java:95)
2021-01-07T11:59:20.1401730Z 	at org.apache.fop.pdf.PDFColorHandler.establishColor(PDFColorHandler.java:88)
2021-01-07T11:59:20.1404243Z 	at org.apache.fop.render.pdf.PDFContentGenerator.setColor(PDFContentGenerator.java:458)
2021-01-07T11:59:20.1418037Z 	at org.apache.fop.render.pdf.PDFContentGenerator.setColor(PDFContentGenerator.java:435)
2021-01-07T11:59:20.1420694Z 	at org.apache.fop.render.pdf.PDFContentGenerator.setColor(PDFContentGenerator.java:460)
2021-01-07T11:59:20.1423341Z 	at org.apache.fop.render.pdf.PDFContentGenerator.updateColor(PDFContentGenerator.java:483)
2021-01-07T11:59:20.1425601Z 	at org.apache.fop.render.pdf.PDFPainter.drawText(PDFPainter.java:428)
2021-01-07T11:59:20.1427539Z 	at org.apache.fop.render.pdf.PDFPainter.drawText(PDFPainter.java:449)
2021-01-07T11:59:20.1429417Z 	at org.apache.fop.render.intermediate.IFRenderer$TextUtil.flush(IFRenderer.java:1238)
2021-01-07T11:59:20.1431596Z 	at org.apache.fop.render.intermediate.IFRenderer.renderText(IFRenderer.java:1064)
2021-01-07T11:59:20.1434114Z 	at org.apache.fop.render.AbstractRenderer.renderInlineArea(AbstractRenderer.java:861)
2021-01-07T11:59:20.1436762Z 	at org.apache.fop.render.intermediate.IFRenderer.renderInlineArea(IFRenderer.java:927)
2021-01-07T11:59:20.1439370Z 	at org.apache.fop.render.AbstractRenderer.renderLineArea(AbstractRenderer.java:845)
2021-01-07T11:59:20.1441714Z 	at org.apache.fop.render.AbstractRenderer.renderBlocks(AbstractRenderer.java:704)
2021-01-07T11:59:20.1443982Z 	at org.apache.fop.render.AbstractRenderer.renderBlock(AbstractRenderer.java:758)
2021-01-07T11:59:20.1446324Z 	at org.apache.fop.render.intermediate.IFRenderer.renderBlock(IFRenderer.java:999)
2021-01-07T11:59:20.1448710Z 	at org.apache.fop.render.AbstractRenderer.renderBlocks(AbstractRenderer.java:689)
2021-01-07T11:59:20.1450920Z 	at org.apache.fop.render.AbstractRenderer.renderFlow(AbstractRenderer.java:591)
2021-01-07T11:59:20.1453700Z 	at org.apache.fop.render.AbstractPathOrientedRenderer.renderFlow(AbstractPathOrientedRenderer.java:789)
2021-01-07T11:59:20.1456742Z 	at org.apache.fop.render.AbstractRenderer.renderMainReference(AbstractRenderer.java:568)
2021-01-07T11:59:20.1459367Z 	at org.apache.fop.render.AbstractRenderer.renderBodyRegion(AbstractRenderer.java:433)
2021-01-07T11:59:20.1527395Z 	at org.apache.fop.render.AbstractRenderer.renderRegionViewport(AbstractRenderer.java:380)
2021-01-07T11:59:20.1530380Z 	at org.apache.fop.render.intermediate.IFRenderer.renderRegionViewport(IFRenderer.java:738)
2021-01-07T11:59:20.1533167Z 	at org.apache.fop.render.AbstractRenderer.renderPageAreas(AbstractRenderer.java:345)
2021-01-07T11:59:20.1535584Z 	at org.apache.fop.render.AbstractRenderer.renderPage(AbstractRenderer.java:318)
2021-01-07T11:59:20.1537894Z 	at org.apache.fop.render.intermediate.IFRenderer.renderPage(IFRenderer.java:587)
2021-01-07T11:59:20.1540155Z 	at org.apache.fop.area.RenderPagesModel.addPage(RenderPagesModel.java:123)
2021-01-07T11:59:20.1543374Z 	at org.apache.fop.layoutmgr.AbstractPageSequenceLayoutManager.finishPage(AbstractPageSequenceLayoutManager.java:316)
2021-01-07T11:59:20.1547116Z 	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.finishPage(PageSequenceLayoutManager.java:243)
2021-01-07T11:59:20.1550506Z 	at org.apache.fop.layoutmgr.PageSequenceLayoutManager.activateLayout(PageSequenceLayoutManager.java:147)
2021-01-07T11:59:20.1553420Z 	at org.apache.fop.area.AreaTreeHandler.endPageSequence(AreaTreeHandler.java:267)
2021-01-07T11:59:20.1555641Z 	at org.apache.fop.fo.pagination.PageSequence.endOfNode(PageSequence.java:139)
2021-01-07T11:59:20.1557598Z 	at org.apache.fop.fo.FOTreeBuilder$MainFOHandler.endElement(FOTreeBuilder.java:362)
2021-01-07T11:59:20.1559323Z 	at org.apache.fop.fo.FOTreeBuilder.endElement(FOTreeBuilder.java:190)
2021-01-07T11:59:20.1561842Z 	at com.sun.org.apache.xml.internal.serializer.ToXMLSAXHandler.endElement(ToXMLSAXHandler.java:263)
2021-01-07T11:59:20.1565284Z 	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:610)
2021-01-07T11:59:20.1569898Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1718)
2021-01-07T11:59:20.1574765Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2883)
2021-01-07T11:59:20.1578704Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
2021-01-07T11:59:20.1582342Z 	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
2021-01-07T11:59:20.1586670Z 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
2021-01-07T11:59:20.1591024Z 	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
2021-01-07T11:59:20.1594161Z 	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
2021-01-07T11:59:20.1596983Z 	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
2021-01-07T11:59:20.1599782Z 	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
2021-01-07T11:59:20.1602652Z 	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
2021-01-07T11:59:20.1605893Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transformIdentity(TransformerImpl.java:660)
2021-01-07T11:59:20.1609513Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:774)
2021-01-07T11:59:20.1612807Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:370)
2021-01-07T11:59:20.1615556Z 	at org.apache.camel.component.fop.FopProducer.transform(FopProducer.java:98)
2021-01-07T11:59:20.1617667Z 	at org.apache.camel.component.fop.FopProducer.process(FopProducer.java:66)
2021-01-07T11:59:20.1683483Z 	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66)
2021-01-07T11:59:20.1707219Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:218)
2021-01-07T11:59:20.1710717Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor$1.process(SharedCamelInternalProcessor.java:112)
2021-01-07T11:59:20.1714390Z 	at org.apache.camel.impl.engine.DefaultAsyncProcessorAwaitManager.process(DefaultAsyncProcessorAwaitManager.java:83)
2021-01-07T11:59:20.1718312Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:109)
2021-01-07T11:59:20.1721484Z 	at org.apache.camel.support.cache.DefaultProducerCache.send(DefaultProducerCache.java:189)
2021-01-07T11:59:20.1724365Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:176)
2021-01-07T11:59:20.1727240Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:172)
2021-01-07T11:59:20.1730360Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.requestBody(DefaultProducerTemplate.java:413)
2021-01-07T11:59:20.1733265Z 	at org.apache.camel.quarkus.component.fop.it.FopResource.post(FopResource.java:55)
2021-01-07T11:59:20.1735871Z 	at org.apache.camel.quarkus.component.fop.it.FopResource_ClientProxy.post(FopResource_ClientProxy.zig:161)
2021-01-07T11:59:20.1737686Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2021-01-07T11:59:20.1739365Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:170)
2021-01-07T11:59:20.1741708Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
2021-01-07T11:59:20.1744639Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:643)
2021-01-07T11:59:20.1748354Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:507)
2021-01-07T11:59:20.1751641Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:457)
2021-01-07T11:59:20.1760478Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.1792558Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:459)
2021-01-07T11:59:20.1795347Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:419)
2021-01-07T11:59:20.1797855Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:393)
2021-01-07T11:59:20.1800386Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:68)
2021-01-07T11:59:20.1803018Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:492)
2021-01-07T11:59:20.1805512Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
2021-01-07T11:59:20.1808059Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
2021-01-07T11:59:20.1864116Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.1868007Z 	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
2021-01-07T11:59:20.1870607Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
2021-01-07T11:59:20.1873394Z 	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:73)
2021-01-07T11:59:20.1876654Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:138)
2021-01-07T11:59:20.1879995Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.access$000(VertxRequestHandler.java:41)
2021-01-07T11:59:20.1883051Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:93)
2021-01-07T11:59:20.1885424Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
2021-01-07T11:59:20.1887498Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2021-01-07T11:59:20.1889394Z 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
2021-01-07T11:59:20.1891613Z 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
2021-01-07T11:59:20.1893316Z 	at java.lang.Thread.run(Thread.java:834)
2021-01-07T11:59:20.1894348Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2021-01-07T11:59:20.1896212Z 	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
2021-01-07T11:59:20.1899159Z 	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
2021-01-07T11:59:20.1901159Z 
2021-01-07T11:59:20.1918934Z 2021-01-07 11:59:20,134 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-1) HTTP Request to /fop/post failed, error id: a430f80f-fc91-40d6-b7d0-f71c6e7a334c-2: org.jboss.resteasy.spi.UnhandledException: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[]
2021-01-07T11:59:20.1923079Z 	at org.jboss.resteasy.core.ExceptionHandler.handleApplicationException(ExceptionHandler.java:106)
2021-01-07T11:59:20.1926022Z 	at org.jboss.resteasy.core.ExceptionHandler.handleException(ExceptionHandler.java:372)
2021-01-07T11:59:20.1928800Z 	at org.jboss.resteasy.core.SynchronousDispatcher.writeException(SynchronousDispatcher.java:218)
2021-01-07T11:59:20.1931507Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:519)
2021-01-07T11:59:20.1934053Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
2021-01-07T11:59:20.1936601Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
2021-01-07T11:59:20.1940542Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.2003273Z 	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
2021-01-07T11:59:20.2005881Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
2021-01-07T11:59:20.2008683Z 	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:73)
2021-01-07T11:59:20.2011934Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:138)
2021-01-07T11:59:20.2015287Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.access$000(VertxRequestHandler.java:41)
2021-01-07T11:59:20.2018407Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:93)
2021-01-07T11:59:20.2021012Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
2021-01-07T11:59:20.2023001Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2021-01-07T11:59:20.2024916Z 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
2021-01-07T11:59:20.2027120Z 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
2021-01-07T11:59:20.2028832Z 	at java.lang.Thread.run(Thread.java:834)
2021-01-07T11:59:20.2029847Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
2021-01-07T11:59:20.2031720Z 	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:519)
2021-01-07T11:59:20.2034702Z 	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
2021-01-07T11:59:20.2037424Z Caused by: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[]
2021-01-07T11:59:20.2039653Z 	at org.apache.camel.support.DefaultExchange.setException(DefaultExchange.java:425)
2021-01-07T11:59:20.2042881Z 	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:69)
2021-01-07T11:59:20.2046719Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:218)
2021-01-07T11:59:20.2050108Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor$1.process(SharedCamelInternalProcessor.java:112)
2021-01-07T11:59:20.2053771Z 	at org.apache.camel.impl.engine.DefaultAsyncProcessorAwaitManager.process(DefaultAsyncProcessorAwaitManager.java:83)
2021-01-07T11:59:20.2057687Z 	at org.apache.camel.impl.engine.SharedCamelInternalProcessor.process(SharedCamelInternalProcessor.java:109)
2021-01-07T11:59:20.2064277Z 	at org.apache.camel.support.cache.DefaultProducerCache.send(DefaultProducerCache.java:189)
2021-01-07T11:59:20.2067663Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:176)
2021-01-07T11:59:20.2071320Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.send(DefaultProducerTemplate.java:172)
2021-01-07T11:59:20.2074443Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.requestBody(DefaultProducerTemplate.java:413)
2021-01-07T11:59:20.2077356Z 	at org.apache.camel.quarkus.component.fop.it.FopResource.post(FopResource.java:55)
2021-01-07T11:59:20.2079886Z 	at org.apache.camel.quarkus.component.fop.it.FopResource_ClientProxy.post(FopResource_ClientProxy.zig:161)
2021-01-07T11:59:20.2081699Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2021-01-07T11:59:20.2083369Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:170)
2021-01-07T11:59:20.2085590Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
2021-01-07T11:59:20.2088513Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:643)
2021-01-07T11:59:20.2092239Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:507)
2021-01-07T11:59:20.2097165Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:457)
2021-01-07T11:59:20.2105080Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
2021-01-07T11:59:20.2109111Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:459)
2021-01-07T11:59:20.2111887Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:419)
2021-01-07T11:59:20.2114392Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:393)
2021-01-07T11:59:20.2116911Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:68)
2021-01-07T11:59:20.2119468Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:492)
2021-01-07T11:59:20.2120891Z 	... 17 more
2021-01-07T11:59:20.2123402Z Caused by: com.oracle.svm.core.jdk.UnsupportedFeatureError: Resource bundle lookup must be loaded during native image generation: com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMessages
2021-01-07T11:59:20.2126730Z 	at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:87)
2021-01-07T11:59:20.2128616Z 	at java.util.ListResourceBundle.loadLookup(ListResourceBundle.java:108)
2021-01-07T11:59:20.2130530Z 	at java.util.ListResourceBundle.handleGetObject(ListResourceBundle.java:130)
2021-01-07T11:59:20.2132268Z 	at java.util.ResourceBundle.getObject(ResourceBundle.java:555)
2021-01-07T11:59:20.2133754Z 	at java.util.ResourceBundle.getString(ResourceBundle.java:521)
2021-01-07T11:59:20.2136367Z 	at com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMsg.getErrorMessage(ErrorMsg.java:311)
2021-01-07T11:59:20.2139904Z 	at com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMsg.toString(ErrorMsg.java:284)
2021-01-07T11:59:20.2142195Z 	at java.lang.String.valueOf(String.java:2951)
2021-01-07T11:59:20.2143175Z 	at java.io.PrintStream.println(PrintStream.java:897)
2021-01-07T11:59:20.2145267Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.error(TransformerImpl.java:1376)
2021-01-07T11:59:20.2148880Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.postErrorToListener(TransformerImpl.java:841)
2021-01-07T11:59:20.2152610Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:782)
2021-01-07T11:59:20.2155917Z 	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:370)
2021-01-07T11:59:20.2158655Z 	at org.apache.camel.component.fop.FopProducer.transform(FopProducer.java:98)
2021-01-07T11:59:20.2160760Z 	at org.apache.camel.component.fop.FopProducer.process(FopProducer.java:66)
2021-01-07T11:59:20.2163826Z 	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66)
2021-01-07T11:59:20.2166018Z 	... 39 more
```

--- a/integration-tests/fop/src/test/java/org/apache/camel/quarkus/component/fop/it/FopIT.java
+++ b/integration-tests/fop/src/test/java/org/apache/camel/quarkus/component/fop/it/FopIT.java
@@ -17,10 +17,8 @@
 package org.apache.camel.quarkus.component.fop.it;
 
 import io.quarkus.test.junit.NativeImageTest;
-import org.junit.jupiter.api.Disabled;
 
 @NativeImageTest
-@Disabled("https://github.com/apache/camel-quarkus/issues/2113")
 class FopIT extends FopTest {
 
 }

